storage-proofs-core/src/settings.rs

```diff
@@ -10,6 +10,7 @@ lazy_static! {
 
 const SETTINGS_PATH: &str = "./rust-fil-proofs.config.toml";
 const PREFIX: &str = "FIL_PROOFS";
+pub const DEFAULT_ROWS_TO_DISCARD: u32 = 2;
```

Comment:
Could this constant be moved to `storage-proofs-core/src/util.rs` where it would be better suited in the `default_rows_to_discard` method instead of the other way around?  Then settings would use that constant

Reply:
Works for me, just a note that then you kind of have a circular dependency. `util` depends on `settings` and `settings` depend on `util`.

Reply:
No, if the constant is moved to util, settings will use the constant and util does not require anything from settings.